packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -341,6 +342,7 @@ function UserMenu({
             })}
           >
             <MenuItem
+              as={userMenu.cloud?.userPreferences ? 'a' : 'button'}
```

Comment:
It'd be nice if we could do the existence check of `userMenu.cloud?` once so we don't bloat the file size.

```
cloudUrls = userMenu.cloud ?? {}
```

Reply:
Going to hold off on updating this until I spend a little more time playing with URLSInterface, but if I leave it as-is then will implement.

Reply:
👍